src/components/Footer/Footer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FormattedHTMLMessage } from 'react-intl';
+import './Footer.css';
+
+const Footer = () => (
+  <footer className="main-footer">
+    <div className="wrapper">
+      <p>
+        <span>
+          <FormattedHTMLMessage id="footer.iso" />
+        </span>
+        <span>
+          &copy; {new Date().getFullYear()} Doppler LLC.{' '}
+          <FormattedHTMLMessage id="footer.allRightReserved" />.{' '}
+          <a className="link--default" href="https://fromdoppler.com/privacidad">
```

Comment:
Is it a good idea using a class for _default link_ in place of assuming that a link without a class is a _default link_?

Reply:
i copy className current use in actual footer. But i can remove this and wait for css library